### PR TITLE
Add two examples of URL() constructor

### PR DESCRIPTION
**Pull request by aleen42**
#### Summary

Add two examples of URL Constructor according to https://github.com/mdn/browser-compat-data/pull/13820.

#### Motivation

To tell readers an edge case when passing empty string value to the first argument.

#### Supporting details

```js
// Chrome, Firefox, Node, New Edge
new URL('', 'http://www.test.com?sid=1').searchParams.get('sid'); // => "1"
// Old Edge
new URL('', 'http://www.test.com?sid=1').searchParams.get('sid'); // => null
```

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/url/url/index.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ let d = new URL('/en-US/docs', b);                     // =
         new URL('http://www.example.com', );           // => 'http://www.example.com/'
         new URL('http://www.example.com', b);          // => 'http://www.example.com/'
 
+        new URL("", "https://example.com/?query=1")    // => 'https://example.com/?query=1' (Edge before 79 removes query arguments)
+        new URL("/a", "https://example.com/?query=1")    // => 'https://example.com/a' (see relative URLs)
```
Comment (sideshowbarker) [inline files/en-us/web/api/url/url/index.md:75]:
```suggestion
        new URL("/a", "https://example.com/?query=1")  // => 'https://example.com/a' (see relative URLs)
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Aleen, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉
